wiki/Hit_Objects/en.md

```diff
@@ -1,68 +1,22 @@
-
-### Slider Speed
-
-A **slider's speed** defines how much time it takes to travel a given distance--or equivalently, how far the slider travels in a given time.
-
-Slider speed is measured in *hundreds of [osupixels](/wiki/Glossary#osupixel) per beat,* so a slider with a speed of 1.00 will travel 100 osupixels (or, 100 pixels at 640x480 resolution) in one full beat. This means that sliders with a particular speed setting will travel faster when the BPM is higher.
-
-Slider speed can be assigned from the Timing tab from the editor main window. You can also assign a multiplier to this speed value by adding an inheriting [timing section](/wiki/Beatmap_Editor/Timing).
-
-It has recently become popular to use slider speeds which are multiples of 0.16. (for instance, 1.60, 1.76, 1.92, etc.) This has the effect of lining up the endpoint of a straight horizontal/vertical slider with osu!'s grid when the slider begins and ends on a whole or half beat, but has its drawbacks when it comes to slightly curved sliders.
-
-### Slider Tick Rate
-
-Slider tick rate is the rate that slider tick appears in slider. For example, slider tick rate 1 means that slider ticks appears once per beat. However, as the slider ticks does not appear in the two end-points of a slider, some slider does not contain any slider tick. For example, a 1-beat slider does not contain slider tick in slider tick rate 1.
-
-Typical slider tick rate includes 1, 2 and 4. 3 is also available but used rarely. In the past slider tick 4 is widely used but slider tick rate 1 and 2 are used more popularly. Slider tick rate 0.5 (slider tick per 2 beat) is also available in the past but it is unrankable now.
+# Hit object
 
-## Spinner
+*Hit objects* are elements which players can interact with during gameplay of a [beatmap](/wiki/Beatmaps/). Each [gamemode](/wiki/Game_Modes) has its own unique set of hit objects which changes the gameplay experience. For a quick overview of each gamemode's hit objects, see the following table:
 
-![A spinner in the default skin.](img/Spinner.jpg "A spinner in the default skin.")
+| osu!standard | osu!taiko | osu!catch | osu!mania |
+| :-: | :-: | :-: | :-: |
+| [hit circles](/wiki/Hit_Objects/Hit_Circle) | hit circles | fruits | notes |
+| [sliders](/wiki/Hit_Objects/Slider) | drumrolls | fruit trails | hold notes|
+| [spinners](/wiki/Hit_Objects/Spinner/en.md) | dendens | bananas | x |
+| x | x | hyperfruits | x |
```

Comment:
this isn't really a hit object as much as a special case of the others. still deserves its own article at some point, but doesn't belong in this table

Reply:
what are you referring to here?